tc-discord-webhooks-server/src/main/java/com/github/playerforcehd/tcdiscordwebhooks/notificator/DiscordNotificator.java

```diff
@@ -298,8 +313,8 @@ public void notifyLabelingFailed(@NotNull Build build, @NotNull VcsRoot vcsRoot,
     @Override
     public void notifyBuildFailing(@NotNull SRunningBuild sRunningBuild, @NotNull Set<SUser> users) {
         String title = "Build is failing";
-        String description = "The build with the ID " + sRunningBuild.getBuildId() + " is failing!";
-        String url = this.sBuildServer.getRootUrl() + "/viewLog.html?buildId=" + sRunningBuild.getBuildId();
+        String description = "The build with the ID " + sRunningBuild.getBuildNumber() + " is failing!";
+        String url = this.sBuildServer.getRootUrl() + "/viewLog.html?buildId=" + sRunningBuild.getBuildNumber();
```

Comment:
Please continue to use `sRunningBuild.getBuildId();` here.
The URL expects the build id. If the build number is customized in the settings, the URL won't work anymore.